### PR TITLE
chore: add client usage documentation to README for Server Side Actions

### PR DESCRIPTION
**Pull request by rdyv**

### CODE REVIEW

**Review by judofyr — DISMISSED** [review]
LGTM. Not sure if other people have opinions here as well.

File: README.md [hunk 1]
```diff
@@ -101,6 +101,15 @@ export async function updateDocumentTitle(_id, title) {
...
   - [Set client configuration](#set-client-configuration)
+  - [Server Side Actions](#server-side-actions)
+    - [Create Action](#create-action)
+    - [ReplaceDraft Action](#replacedraft-action)
+    - [Edit Action](#edit-action)
+    - [Delete Action](#delete-action)
+    - [Discard Action](#discard-action)
+    - [Publish Action](#publish-action)
+    - [Unpublish Action](#unpublish-action)
+    - [Action options](#action-options)
```
Comment (rneatherway) [inline README.md]: Please could you put these in alphabetical order to match the documentation? (https://www.sanity.io/docs/http-actions)

**Review by rneatherway — CHANGES_REQUESTED** [review]
Please use alphabetical order to match the API docs.

File: README.md (hunk 1 above)
Comment (rdyv) [reply README.md]: Reordered items in order.

**Review by rneatherway — APPROVED** [review]

**Pull request merged**